Tests/MapboxMapsTests/Location/MockLocationStyleDelegate.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+@testable import MapboxMaps
+
+class MockLocationStyleDelegate: LocationStyleDelegate {
+    func addLayer(_ layer: Layer, layerPosition: LayerPosition?) throws {}
+    func removeLayer(withId id: String) throws {}
+    func layerExists(withId id: String) -> Bool { return false }
+    func setLayerProperties(for layerId: String, properties: [String: Any]) throws {}
+    func addSource(_ source: Source, id: String) throws {}
+    func removeSource(withId id: String) throws {}
+    func setSourceProperty(for sourceId: String, property: String, value: Any) throws {}
+
+    //swiftlint:disable function_parameter_count
+    func addImage(_ image: UIImage, id: String, sdf: Bool, stretchX: [ImageStretches], stretchY: [ImageStretches], content: ImageContent?) throws {}
+}
```

Comment:
Could we add stubs (`Stub<>`) here? Or add a ticket to add stubs as part of the location re-design that @neelmistry94  is doing?

Reply:
I will add it as part of my work

Reply:
thank you!